packages/calling/src/CallingClient/calling/types.ts

```diff
@@ -229,9 +231,15 @@ export type CallErrorEmitterCallBack = (error: CallError) => void;
 export type RetryCallBack = (interval: number) => void;
 
 export interface ICallManager extends Eventing<CallEventTypes> {
-  createCall: (destination: CallDetails, direction: CallDirection, deviceId: string) => ICall;
+  createCall: (
+    destination: CallDetails,
+    direction: CallDirection,
+    deviceId: string,
+    lineId: string
+  ) => ICall;
   endCall: (callId: CallId) => void;
   getCall: (callId: CallId) => ICall;
   updateActiveMobius: (url: string) => void;
   getActiveCalls: () => Record<string, ICall>;
+  addLine: (deviceId: string, line: ILine) => void;
```

Comment:
updateLine would be a better name for this internal function

Reply:
Changed the name